Header header
string color
uint32 position_x
uint32 position_y

